packages/account-tree-controller/src/AccountTreeController.ts

```diff
@@ -1072,7 +1174,7 @@ export class AccountTreeController extends BaseController<
 
     this.update((state) => {
       // Update persistent metadata
-      state.accountGroupsMetadata[groupId] ??= {};
+      state.accountGroupsMetadata[groupId] = {};
```

Comment:
Wait, why are we changing this? We could overwrite other metadata because of this no?